app/templates/404.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```

Comment:
Would it be possible to use the `{% extends 'base.html' %}` syntax to avoid rewriting the first 20 lines?

Reply:
Actually, I'm not sure that's a good idea. Maybe we should make it so that the navbar doesn't appear in the 404. What do you think?

Reply:
I don't think having a navbar would look good on the 404 page, however maybe we could just put a simple link back to the home page. Thoughts?

Reply:
Great idea let's do that